service/defaultcomponents/components.go

```diff
@@ -51,6 +52,7 @@ func Factories() (otelcol.Factories, error) {
 		awsappsignals.NewFactory(),
 		batchprocessor.NewFactory(),
 		cumulativetodeltaprocessor.NewFactory(),
+		filterprocessor.Newfactory(),
```

Comment:
You'll need to update the unit test for this to account for the new processor.

Reply:
Nice catch, we'll update this